classes/TranslatableBehavior.php

```diff
@@ -58,15 +59,25 @@ public function __construct($model)
 
         $this->initTranslatableContext();
 
-        $this->model->bindEvent('model.beforeGetAttribute', function($key) {
-            if ($key !== 'translatable' && $this->isTranslatable($key)) {
-                return $this->getAttributeTranslated($key);
+        $this->model->bindEvent('model.beforeGetAttribute', function($key) use ($model) {
+            if ($this->isTranslatable($key)) {
+                $value = $this->getAttributeTranslated($key);
+                if ($model->hasGetMutator($key)) {
+                    $method = 'get'.Str::studly($key).'Attribute';
+                    $value = $model->{$method}($value);
+                }
+                return $value;
             }
         });
 
-        $this->model->bindEvent('model.beforeSetAttribute', function($key, $value) {
-            if ($key !== 'translatable' && $this->isTranslatable($key)) {
-                return $this->setAttributeTranslated($key, $value);
+        $this->model->bindEvent('model.beforeSetAttribute', function($key, $value) use ($model) {
```

Comment:
@mjauvin Space between `function` and `(`.
```suggestion
        $this->model->bindEvent('model.beforeSetAttribute', function ($key, $value) use ($model) {
```